freebasics/templates/custom_css.html

```diff
@@ -0,0 +1,21 @@
+<style type="text/css">
```

Comment:
@codiebeulaine this style comes from [this template](https://github.com/projectmushroom/fb-admin/blob/master/dist/css/skins/template-base.css#L156-L175), you can see the preview [here](http://projectmushroom.com/fb_admin_v1.7/export_no-card.html)